### PR TITLE
Feature/cluster group rename

### PR DESCRIPTION
**Pull request by svt-kmcdaniel**
### Description
Implemented the Cluster Group rename Post Operation

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/cluster_groups.py [hunk 1]
```diff
@@ -75,3 +75,13 @@
 cluster_group = cluster_groups.get_by_name(cluster_group_name)
 print(f"{cluster_group}")
 print(f"{pp.pformat(cluster_group.data)} \n")
+
+cluster_group_name = cluster_group_object.data["name"]
+print("\n\nrename")
+cluster_group = cluster_group_object
```
Comment (sijeesh) [inline examples/cluster_groups.py]: Do we need to assign cluster object to a different variable?

**Review by sijeesh — CHANGES_REQUESTED** [review]

File: examples/cluster_groups.py (hunk 1 above)
Comment (svt-kmcdaniel) [reply examples/cluster_groups.py]: It doesn't, it was just to keep the code looking the same from all the other steps.  

I removed the line and updated the references below to cluster_group

**Review by sijeesh — APPROVED** [review]
LGTM

**Comment by sijeesh** [comment]
@resolve conflicts.

**Comment by svt-kmcdaniel** [comment]
> @resolve conflicts.

Conflict Resolved

**Comment by svt-kmcdaniel** [comment]
resolved conflicts, again.

**Comment by svt-kmcdaniel** [comment]
@sijeesh this PR looks ready, has no conflicts with the base branch, and appears to be ready for merge.

**Pull request merged**